src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? filterPeopleBySex(people, 'm')
+      .filter(person => century === Math.ceil(person.died / 100))
+    : filterPeopleBySex(people, 'm');
```

Comment:
Good: you've created a utility function.
Bad: you've created two loops instead of one. Imagine if you have a huge array of data. It can be hard to process.